### PR TITLE
Various bug fixes since 1.0.0

### PR DESCRIPTION
**Pull request by tlunter**
@nahiluhmot

These were fixed and tested during actual deploys.  Each commit is distinct.

### CODE REVIEW

File: lib/aerosol/instance.rb [hunk 1]
```diff
@@ -22,6 +22,14 @@ def private_ip_address
     description[:private_ip_address]
   end
 
+  def address
```
Comment (nahiluhmot) [inline lib/aerosol/instance.rb]: Do we want to fallback to the private ip or the public hostname? Or should it be config?

Comment (nahiluhmot) [inline lib/aerosol/instance.rb]: Also since we have `activesupport`, you might as well use `blank?` here.

**Comment by nahiluhmot** [comment]
1 nitpick, looks good otherwise.

File: lib/aerosol/instance.rb (hunk 1 above)
Comment (tlunter) [inline lib/aerosol/instance.rb]: We've always used the public hostname falling back to the private ip.

Comment (nahiluhmot) [inline lib/aerosol/instance.rb]: Sounds good, :+1:

**Pull request merged**